src/main/java/com/minecolonies/coremod/achievements/ModAchievements.java

```diff
@@ -39,279 +38,267 @@
-    public static final Achievement achievementHarvest1000Wheat    = new MineColoniesAchievement("farmer.wheatHarvested.1000", 12, 17, Blocks.HAY_BLOCK,
-                                                                                                  achievementHarvest500Wheat).registerStat();
+    public static final MineColoniesAchievement achievementBuild1000Huts       = new MineColoniesAchievement("builder.hutsBuilt.1000", 12, 3, ModBlocks.blockHutCitizen,
+            achievementBuild500Huts);
+    public static final MineColoniesAchievement achievementCatch1000Fish       = new MineColoniesAchievement("fisher.fishCaught.1000", 12, 5, Items.FISH,
+            achievementCatch500Fish);
+    public static final MineColoniesAchievement achievementKill1000Mobs        = new MineColoniesAchievement("guard.mobsKilled.1000", 12, 7, Items.ENDER_EYE,
+            achievementKill500Mobs);
+    public static final MineColoniesAchievement achievementMine1000Ores        = new MineColoniesAchievement("miner.oresMined.1000", 12, 9, Blocks.LAPIS_ORE,
+            achievementMine500Ores);
+    public static final MineColoniesAchievement achievementMine1000Diamonds    = new MineColoniesAchievement("miner.diamondsMined.1000", 12, 10, Items.DIAMOND,
+            achievementMine500Diamonds);
+    public static final MineColoniesAchievement achievementFell1000Trees       = new MineColoniesAchievement("lumberjack.treesFelled.1000", 12, 12, Blocks.LOG,
+            achievementFell500Trees);
+    public static final MineColoniesAchievement achievementPlant1000Saplings   = new MineColoniesAchievement("lumberjack.saplingsPlanted.1000", 12, 13, Blocks.SAPLING,
+            achievementPlant500Saplings);
+    public static final MineColoniesAchievement achievementHarvest1000Carrots  = new MineColoniesAchievement("farmer.carrotsHarvested.1000", 12, 15, Items.GOLDEN_CARROT,
+            achievementHarvest500Carrots);
+    public static final MineColoniesAchievement achievementHarvest1000Potatoes = new MineColoniesAchievement("farmer.potatoesHarvested.1000", 12, 16, Items.POTATO,
+            achievementHarvest500Potatoes);
+    public static final MineColoniesAchievement achievementHarvest1000Wheat    = new MineColoniesAchievement("farmer.wheatHarvested.1000", 12, 17, Blocks.HAY_BLOCK,
+            achievementHarvest500Wheat);
 
     // Sizes
     /**
      * Reach {@link ModAchievements#ACHIEVEMENT_SIZE_SETTLEMENT} citizens.
      */
-    public static final Achievement achievementSizeSettlement = new MineColoniesAchievement("size.settlement", 4, 1, ModItems.itemAchievementProxySettlement,
-                                                                                             achievementTownhall).registerStat();
+    public static final MineColoniesAchievement achievementSizeSettlement = new MineColoniesAchievement("size.settlement", 4, 1, ModItems.itemAchievementProxySettlement, achievementTownhall);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 191 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00103)